packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -77,8 +77,8 @@ export function FormatFile({
       >
         <div className={thumbnailContainerClassNames}>
           <InternalThumbnail
-            name={file.name}
-            hideName={display === "compact"}
+            key={file.key}
```

Comment:
Used the `key` to tell React to rerender this when it's a different key rather than writing the most elaborate `useEffect` that gives us the same result